arm-compute/2015-11-01-preview/swagger/containerService.json

```diff
@@ -0,0 +1,516 @@
+            "count": {
+              "type": "integer",
+              "format": "int32",
+              "maximum": 100,
+              "minimum": 1,
+              "description": "No. of agents (VMs) that will host docker containers"
+            },
+            "vmSize": {
+              "type": "string",
+              "enum": [
+                "Standard_A0",
+                "Standard_A1",
+                "Standard_A2",
+                "Standard_A3",
+                "Standard_A4",
+                "Standard_A5",
+                "Standard_A6",
+                "Standard_A7",
+                "Standard_A8",
+                "Standard_A9",
+                "Standard_A10",
+                "Standard_A11",
+                "Standard_D1",
+                "Standard_D2",
+                "Standard_D3",
+                "Standard_D4",
+                "Standard_D11",
+                "Standard_D12",
+                "Standard_D13",
+                "Standard_D14",
+                "Standard_D1_v2",
+                "Standard_D2_v2",
+                "Standard_D3_v2",
+                "Standard_D4_v2",
+                "Standard_D5_v2",
+                "Standard_D11_v2",
+                "Standard_D12_v2",
+                "Standard_D13_v2",
+                "Standard_D14_v2",
+                "Standard_G1",
+                "Standard_G2",
+                "Standard_G3",
+                "Standard_G4",
+                "Standard_G5",
+                "Standard_DS1",
+                "Standard_DS2",
+                "Standard_DS3",
+                "Standard_DS4",
+                "Standard_DS11",
+                "Standard_DS12",
+                "Standard_DS13",
+                "Standard_DS14",
+                "Standard_GS1",
+                "Standard_GS2",
+                "Standard_GS3",
+                "Standard_GS4",
+                "Standard_GS5"
+              ],
+              "description": "Size of agent VMs"
+            },
+            "dnsPrefix": {
+              "type": "string",
+              "description": "DNS prefix to be used to create FQDN for this agent pool"
+            },
+            "fqdn": {
+              "readOnly": true,
+              "type": "string",
+              "description": "FDQN for the agent pool"
+            }
+          },
+          "required": [
+            "name",
+            "dnsPrefix"
+          ],
+          "description": ""
+        },
+        "ContainerServiceWindowsProfile": {
+          "properties": {
+            "adminUsername": {
+              "type": "string",
+              "description": "The administrator username to use for Windows jumpbox"
+            },
+            "adminPassword": {
+              "type": "string",
+              "description": "The administrator password to use for Windows jumpbox"
+            }
+          },
+          "required": [
+            "adminUsername",
+            "adminPassword"
+          ],
+          "description": ""
```

Comment:
Please provide proper description. This will translate to client lib and msdn documentation.